pkg/upgrade/upgrade.go

```diff
@@ -222,6 +239,39 @@ func (u *Upgrade) ensureManagedByLabelOnDBNamespaces(ctx context.Context) error
 	return nil
 }
 
+// This is needed only for 0.10 to 0.11 upgrade.
+// TODO: Remove after 0.11.0 release.
```

Comment:
Do we really want to remove this after 0.11? Users may use later versions of the CLI (e.g. 1.0.0) to do an upgrade from 0.10 to 0.11. Maybe we should keep the code (at least for a few more releases) and use it conditionally (see the other comments above)

Reply:
Ok, I hadn't considered the fact that users can use a newer CLI to install/upgrade older versions..

I've updated the code so that it runs conditionally (based on version), and also removed the TODOs...